src/schemas/parallel.json

```diff
@@ -8,13 +8,10 @@
       "enum": ["Parallel"]
     },
     "Parameters": {
-      "type": ["object", "array"],
-      "items": {
-        "type": "object"
-      }
+      "$ref": "paths.json#/definitions/asl_payload_template"
```

Comment:
same parameters issue here.

do we need to support array here? If so, the items should be the new paths PayloadTemplate shape